### PR TITLE
Remove DellEMC collections from community.general collectons

### PR DESCRIPTION
**Pull request by rajeevarakkal**
##### SUMMARY
Need to get removed the dellemc collections community general repository as dellemc all modules are now available in Galaxy `dellemc.openmanage` collection.

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below
Already a PR created to update the botmeta & runtime files 
https://github.com/ansible/ansible/pull/71875
```

### CODE REVIEW

**Review by felixfontein — CHANGES_REQUESTED** [review]
Adding negative review to prevent premature merge. `dellemc.openmanage` is not included in Ansible (yet), and we haven't decided how to proceed with such removals yet.

**Comment by rajeevarakkal** [comment]
> Adding negative review to prevent premature merge. `dellemc.openmanage` is not included in Ansible (yet), and we haven't decided how to proceed with such removals yet.

Hi @felixfontein , are you referring to this PR https://github.com/ansible/ansible/pull/71875 completion?

**Comment by felixfontein** [comment]
@rajeevarakkal no, I'm referring to the 'move content between collections where the target collection is not part of Ansible' policy which hasn't been decided yet.

Also, I don't think that PR in ansible/ansible will get merged, since ansible_builtin_runtime.yml is essentially frozen.

**Comment by felixfontein** [comment]
See https://github.com/ansible/community/issues/539#issuecomment-693619299, "to discuss: what happens when the target collection is (not yet) contained in Ansible?".

**Comment by rajeevarakkal** [comment]
@felixfontein , i didn't get exactly target collection  not `contained in ansible`, yes this `dellemc.openmanage` collections are not ansible certified, is that what you mean?

**Comment by felixfontein** [comment]
No, I mean Ansible itself (https://pypi.org/project/ansible/) - it was formerly called ACD (Ansible Community Distribution), maybe that name is more familiar to you. The collections included in Ansible 2.10 are listed here: https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in

**Comment by rajeevarakkal** [comment]
@felixfontein , Thanks, will go through this once.

**Comment by gundalow** [comment]
`meta/runtime.yml` needs updating for every plugin removed. An entry needs adding to point to the new collection. 
`.github/BOTMETA.yml` also needs updating to remove the entries about these files and directories.

**Comment by rajeevarakkal** [comment]
@gundalow , The three contributed modules from dellemc -ie `remote_managment/dellemc` are now we are able to publish as part of 'dellemc.openmanage' collections in the ansible galaxy space.

There been a request/communicaton earlier that 'community.general' is temporary space and teams need to host their collections in to Galaxy space or other repository.

Since we have our collections available at 'dellemc.openmanage', what should we do, what next?

**Comment by thedoubl3j** [comment]
@rajeevarakkal you still need to update the files @gundalow referenced before this can get merged

**Comment by felixfontein** [comment]
There seems to be no modules called `dellemc_idrac` and `ome` in the new collection, but you remove these here. Have they been renamed? And are the other modules compatible to their verisons in the DellEMC collection?

**Comment by rajeevarakkal** [comment]
@felixfontein , yes these module utils are present in `dellemc.openmanage` collection https://galaxy.ansible.com/dellemc/openmanage.

I will be making the change suggested by @gundalow in few days and update the PR.

**Comment by felixfontein** [comment]
@rajeevarakkal thanks, I now found them, I looked in the wrong directory :) (I was looking at the git repository)

**Comment by felixfontein** [comment]
For the changes of BOTMETA and meta/runtime.yml, you can take a look at https://github.com/ansible-collections/community.general/pull/1304/files

**Comment by felixfontein** [comment]
For an example of how this PR should look like, see both f896c2986c024436a9e12ea6f9e5bbc828907dc5 (main removal) and dbae7da6bcfeea9c1c64060b577dec72b8f9f2a0 (fixes changelog fragment, removes changelog fragments).

Please note that this can only be merged once the new collection is included in Ansible 2.10.

**Comment by felixfontein** [comment]
@rajeevarakkal could you please update/adjust this PR in a similar manner to https://github.com/ansible-collections/community.general/commit/e53f153e3071d94174036662dcba63e628a6d7a5 ? The removal would happen in community.general 3.0.0. Once this PR is in a good shape and ansible-community/ansible-build-data#58 has been merged, we can merge it and add an announcement to the next community.general 2.x.y release announcing this removal. (And of course update ansible/ansible#73046.)

**Comment by felixfontein** [comment]
@rajeevarakkal the PR now has conflicts, could you please rebase it?

File: changelogs/fragments/948-dellemc-migration-removal.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+removed_features:
+- >
+  ``ome_device_info``,  ``idrac_firmware`` and  ``idrac_server_config_profile`` modules have been removed from this collection.
```
Comment (felixfontein) [inline changelogs/fragments/948-dellemc-migration-removal.yml:3]: Can you make sure that the fragment has the exact same form as in https://github.com/ansible-collections/community.general/commit/e53f153e3071d94174036662dcba63e628a6d7a5#diff-def1d3c6591c31e8b12257385e6f6cc6155a43ec2acc8bded4a62b8fca4cbf8eR1? Thanks.

**Comment by rajeevarakkal** [comment]
> @rajeevarakkal the PR now has conflicts, could you please rebase it?

on it. Resolving the same

File: changelogs/fragments/948-dellemc-migration-removal.yml (hunk 1 above)
Comment (rajeevarakkal) [reply changelogs/fragments/948-dellemc-migration-removal.yml:3]: sure

Comment (rajeevarakkal) [reply changelogs/fragments/948-dellemc-migration-removal.yml]: this is done

**Comment by felixfontein** [comment]
@rajeevarakkal could you please do a proper rebase to remove the merge commits, and remove all changes from this PR that do not belong in it (like the changes to .azure-pipelines/ and test/utils/)? Thanks.

**Comment by rajeevarakkal** [comment]
> uld you please do a proper rebase to remove the merge commits,

Looks like rebase is running into chained conflicts. I will go ahead nd create a fresh PR and provide the link here. Is that fine?

**Comment by felixfontein** [comment]
@rajeevarakkal that's fine as well!

**Comment by rajeevarakkal** [comment]
> @rajeevarakkal that's fine as well!

@felixfontein here is the new PR https://github.com/ansible-collections/community.general/pull/1699

**Comment by rajeevarakkal** [comment]
New PR created due to multiple conflict here - https://github.com/ansible-collections/community.general/pull/1699

**Pull request closed without merging**